packages/DraftEditor/src/interfaces/index.tsx

```diff
@@ -0,0 +1,175 @@
+import React, { FunctionComponent } from "react";
+
+import {
+  ContentBlock,
+  ContentState,
+  DraftBlockRenderMap,
+  DraftBlockType,
+  DraftDragType,
+  DraftEditorCommand,
+  DraftHandleValue,
+  DraftInlineStyle,
+  DraftStyleMap,
+  EditorState,
+  SelectionState
+} from "draft-js";
+
+export type DraftTextAlignment = "left" | "center" | "right";
```

Comment:
I think you have constants saved you could use here

Reply:
CENTER | LEFT... refers to a value, can't be used as a type.